### ISSUE TITLE
Randomise the races.

### ISSUE THREAD

**Issue by jon5791** [HashLips/generative-art-opensource#141]
Is there a way to randomise the races so it doesn't do them all in a row. I don't want to have 1 - 1000 be only one race and then have 1001 - 2000 be the next race.

**Comment by jon5791** [comment]
Forgot to mention I am using branch 4

**Comment by NetworkEntity** [comment]
Would also like to know a way to do this

**Comment by SDM-Shroom** [comment]
I'm also curious about this

**Comment by tdr357** [comment]
const getRace = (_editionCount) => {
  let race = "No Race";
  if (Math.random() > 0.3) {
      race = "HumanMale"
} else {
  race = "HumanFemale"
}
  return race;
};



PS: If you have code to stop overlapping layers eg: IF Mask THEN No CIG. I'd appreciate some help in return! The code i've supplied will be placed in Index.js :)

**Comment by jon5791** [comment]
Thank you so much for the code I added the ability to have as many races as you want:

const getRace = (_editionCount) => {
  let race = "No Race";
// max is the amount of races you need //
if (Math.floor(Math.random() * (max - 1+ 1)) + min == 1){
  race = "HumanMale"
  }
else if (Math.floor(Math.random() * (max - 1+ 1)) + min == 2){
  race = "HumanFemale"
  } 
else if (Math.floor(Math.random() * (max - 1+ 1)) + min == 3){
  strain = "Alien"
  } 
else if (Math.floor(Math.random() * (max - 1+ 1)) + min == 4){
  strain = "Martian"
  } 
return race;
};

What are you trying to do with the layers that is making them overlap?

**Comment by jon5791** [comment]
> const getRace = (_editionCount) => {
> let race = "No Race";
> if (Math.random() > 0.3) {
> race = "HumanMale"
> } else {
> race = "HumanFemale"
> }
> return race;
> };
> 
> PS: If you have code to stop overlapping layers eg: IF Mask THEN No CIG. I'd appreciate some help in return! The code i've supplied will be placed in Index.js :)

Not sure why you are getting layers overlapping have you changed the original code on layer building?

**Comment by tdr357** [comment]
No, have you got a link to the video, might of missed that. I want an IF statement which cancels layer building if a mask is supplied first

**Comment by tdr357** [comment]
> > const getRace = (_editionCount) => {
> > let race = "No Race";
> > if (Math.random() > 0.3) {
> > race = "HumanMale"
> > } else {
> > race = "HumanFemale"
> > }
> > return race;
> > };
> > PS: If you have code to stop overlapping layers eg: IF Mask THEN No CIG. I'd appreciate some help in return! The code i've supplied will be placed in Index.js :)
> 
> Not sure why you are getting layers overlapping have you changed the original code on layer building?

read above brotha

**Comment by calebharris216** [comment]
> const getRace = (_editionCount) => { let race = "No Race"; if (Math.random() > 0.3) { race = "HumanMale" } else { race = "HumanFemale" } return race; };
> 
> PS: If you have code to stop overlapping layers eg: IF Mask THEN No CIG. I'd appreciate some help in return! The code i've supplied will be placed in Index.js :)

Hey, where does this code go? Do I have to remove the race weights? Is there anything else I need to do? I'm assuming I replace HumanMale and HumanFemale with my two races? would appreciate a quick reply
